hugo/content/docs/tutorials/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Tutorials
```

Comment:
What do you think about the following layout:

* Tutorial (singular)
  * `0. Introduction` (optional, if we need it)
  * `1. Storing and Versioning Data`
  * `2. Data Pipelines and Reproducibility` (TBA)
  * `3. Remote Storage` (TBA)

Reply:
On second thought, I think having an `Introduction` section would be a good idea right off the bat. We can introduce the project, but also put all the setup stuff in there -- Python venv, installing Dud, etc.